internal/config/config.go

```diff
@@ -634,6 +644,11 @@ func checkEnvVars(ctx context.Context, cfg *Configuration) {
 		cfg.AutoPublishingToOnChainRHS = common.ToPointer(true)
 	}
 
+	if cfg.MediaTypeManager.StrictMode == nil {
+		log.Info(ctx, "ISSUER_MEDIA_TYPE_MANAGER_STRICT_MODE is missing and the server set up it as true")
```

Comment:
server set up it as true? how is that possible? why do we need it? default value, which can be overwritten by config.